CMakeLists.txt

```diff
@@ -336,6 +333,24 @@ endif()
 check_function_exists(chmod HAVE_CHMOD)
 check_function_exists(umask HAVE_UMASK)
 
+if(USE_PTHREADS)
+	check_include_file(pthread.h HAVE_PTHREAD_H)
+	if(HAVE_PTHREAD_H)
```

Comment:
You should disable USE_PTHREADS if pthread.h is missing instead of defining a new variable.